ios/NEMCatapultWallet/Info.plist

```diff
@@ -89,5 +89,7 @@
 	<string>Light</string>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>ITSAppUsesNonExemptEncryption</key>
+    <false/>
```

Comment:
Are you sure this is valid? Mobile wallet supports message encryption. Is there a list of exempted algorithms somewhere?

Reply:
Positive. It was already set to false in the current and previously released versions. Probably was being set manually in the TestFlight. See the latest app released info below.

![image](https://user-images.githubusercontent.com/6256269/139851980-6b0f7352-68c3-45af-85ab-307afbff5c7e.png)

Reply:
Can you explain to me why this is right? 

The exemptEncryption flag is to do with cryptography that is exempt from the export encryption list: https://developer.apple.com/documentation/bundleresources/information_property_list/itsappusesnonexemptencryption


We seem to be using CryptoJS to perform the encryption in the SDK.

Reply:
Okay, I see what you mean now. According to the link you provided, I think your question is valid.

I basically translated what's done manually into code(CICD pipeline). This flag was never set in the repository, therefore, the CICD was failing and when I checked the app store saw that it was historically always set to false (for all of the previous builds) in the TestFlight; without setting that flag it is giving a `compliance error` and forcing you to manually set it. That's why I set it to false in the repo.

Already asked @cryptoBeliever and @rg911  but no one seems to be knowing the background about this one.

If we set it to _true_ we won't be able to publish to TestFlight and will be blocked until the process is complete. And the process will be, we'll need to prepare a compliance document, we'll be undergoing review and not sure how long it will take and there is a risk of rejection.

Not sure how to proceed, do you have any ideas on this one @0x6861746366574 ?

Related read:
https://help.apple.com/app-store-connect/#/dev63c95e436
https://help.apple.com/app-store-connect/#/devc3f64248f
https://bis.doc.gov/index.php/documents/new-encryption/1652-cat-5-part-2-quick-reference-guide/file
https://stackoverflow.com/questions/35739361/itsappusesnonexemptencryption-export-compliance-while-internal-testing

Reply:
There are exemption criteria listed here too:

![image](https://user-images.githubusercontent.com/17684281/140287705-1b9b09c8-7a75-47a6-ba4a-3e506f71cef8.png)

From Test Flight:
```
Make sure that your app meets the criteria of the exemption listed below. You are responsible for the proper classification of your product. Incorrectly classifying your app may lead to you being in violation of U.S. export laws and could make you subject to penalties, including your app being removed from the App Store.

You can select Yes for this question if the encryption of your app is:
(a) Specially designed for medical end-use
(b) Limited to intellectual property and copyright protection
(c) Limited to authentication, digital signature, or the decryption of data or files
(d) Specially designed and limited for banking use or “money transactions”; or
(e) Limited to “fixed” data compression or coding techniques

You can also select Yes if your app meets the descriptions provided in Note 4 for Category 5, Part 2 of the U.S. Export Administration Regulations.
```

Reply:
the wallet is using aes-256-gcm [https://github.com/symbol/symbol-sdk-typescript-javascript/blob/dev/src/core/crypto/Crypto.ts#L24]

since that is supported in Apple's crypto kit [https://developer.apple.com/documentation/cryptokit/aes/gcm] we should be fine

it was clarified to me that `Your app uses encryption limited to that within the Apple operating system` allows any implementation(s) of references algorithms, not just the Apple OS implementation